manager/testdata/new-blueprints/scenario2/plotter.yaml

```diff
@@ -72,12 +72,6 @@ spec:
           parameters:
             source:
               assetId: "m4d-notebook-sample/paysim"
-            api:
```

Comment:
Not clear why you are removing the api from the plotter.  We removed it from the blueprint because it's not necessary for deployment.  However, the plotter should contain everything needed to understand the flow of data between the components.  We need to include via what (data source, api, ...) the data may be accessed/written

Reply:
@simanadler can you please elaborate why it's not needed for the deployment? The deployment needs to know the protocol of the service that e.g. a module exposes. (Capabilities in a module are an array and thus multiple different interfaces could be supported by one module)

Reply:
The deployment needs to know the chart image and the values to use. This information is passed via module arguments.

Reply:
Agreed. A chart and it's values are in the end needed for the deployment. My question is where are these values (that say it's an arrow-flight service) coming from? If they are not defined in the plotter how should they be put into the helm chart?

Reply:
These values have been defined in the asset list as a new type of connection. According to this approach, arrow-flight service does not declare how it can be accessed but rather any module that needs to access the arrow-flight service defines what it needs: "data source: format = arrow, protocol=arrow-flight, endpoint=arrow-flight-service:80, assetID=user-asset".